examples/device_discovery.py

```diff
@@ -0,0 +1,27 @@
+try:
+    from o2x5xx import DiscoveryClient
```

Comment:
This needs to be `from o2x5xx.device.discovery import DiscoveryClient`

Reply:
The import statement in source/device/__init__.py for the discovery Client was missing:
from .discovery import *
Fixed!